internal/watcher/watcher_test.go

```diff
@@ -0,0 +1,212 @@
+package watcher
+
+import (
+	"crypto/md5" //nolint:gosec
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetAllInDirectory(t *testing.T) {
+	const (
+		filesMultiplier    = 10
+		expectedDirCount   = 2
```

Comment:
```suggestion
		expectedDirsCount  = 2
```

Reply:
(this is not really important, so it's approved and up to you to apply).

Reply:
My mistake